thoth/package_extract/core.py

```diff
@@ -48,6 +50,31 @@ def extract_buildlog(input_text: str) -> typing.List[dict]:
     return result
 
 
+def get_cuda_version() -> str:
```

Comment:
```python
from typing import Optional

def get_cuda_version() -> Optional[str]:
```


This is possibly issue also in thamos.

Reply:
Thanks, I'll do here and there.

Reply:
Overall, do you think this is OK way of finding the cuda version? For me, it looks good.

Reply:
I'm not sure here (if the binary used is guaranteed to be present), worth some testing.